app/shared/productTypes.tsx

```diff
@@ -154,8 +159,8 @@ export const shouldHaveHolidayStopsFlow = (
 export const createProductDetailFetcher = (
   productType: ProductType,
   subscriptionName?: string
-) => async () =>
-  await fetch(
+) => () => {
+  const result = fetch(
```

Comment:
there's no need for a `const` here can just be an expression returning the `fetch(...)`